tests/config/pubsub/kafka/pubsub.yml

```diff
@@ -10,5 +10,7 @@ spec:
     value: localhost:9092
   - name: consumerID
```

Comment:
We can remove this meta property now. Correct?

Reply:
yes